bitnami/memcached/templates/service.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if eq .Values.architecture "high-availability" }}
+  clusterIP: None
+  {{- end }}
```

Comment:
How can I access Memcached when using "high-availability" architecture? 
We're using the service as a headless svc when using a statefulset, but there's no other alternative service. So I cannot configure my app to use it and I my requests are not balanced between the replicas.

Reply:
When running several memcache-servers you'd make them all known to the client, which uses consistent-hashing to route directly to the right instance: https://www.mikeperham.com/2009/01/14/consistent-hashing-in-memcache-client/

OR you stick mcrouter in front: https://github.com/facebook/mcrouter/wiki/Replicated-pools-setup in front, to let it deal with the hashing and/or replication/routing.

Reply:
It makes sense. Should we document this in the README.md? 
We should also adapt the NOTES.txt since it says:

```
Memcached can be accessed on port 11211 on the following DNS name from within your cluster: {{ template "memcached.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
```

That won't be true when using "high-availability" architecture

Reply:
PTAL